src/lancode.c

```diff
@@ -105,58 +92,66 @@ int resolveService(const char *service) {
     return port;
 }
 
-int lanrecv_init(void) {
-    if (lan_active == 0)
-	return (1);
+int lan_recv_init(void) {
+    int lan_bind_rc;
+    long lan_save_file_flags;
+    struct sockaddr_in lan_sin;
+
+    if (!lan_active)
+	return 0;
 
     sprintf(default_lan_service, "%d", lan_port);
     bzero(&lan_sin, sizeof(lan_sin));
     lan_sin.sin_family = AF_INET;
     lan_sin.sin_addr.s_addr = htonl(INADDR_ANY);
     lan_sin.sin_port = htons(resolveService(default_lan_service));
```

Comment:
just a side note: I find using `resolveService()` (i.e. `getservbyname()`) an overkill as no-one will use a non-privileged UDP service (port) by its name for intra-Tlf communication.
In other IP-related cases we just use plain numeric port, e.g.
```
address.sin_port = htons(netkeyer_port);
```

Reply:
You are right. It just got inherited from Rein. We should clean it out in future work.